policyengine_us/variables/household/demographic/person/total_college_hours.py

```diff
@@ -0,0 +1,9 @@
+from policyengine_us.model_api import *
+
+
+class total_college_hours(Variable):
```

Comment:
I may have to create a sc_semesters_hours, according to the tax form instruction, 
![image](https://github.com/PolicyEngine/policyengine-us/assets/113824411/3ee8d5c9-ed5d-474f-a7b9-045a3f5e1aea)
but the money you spend on summer hours could count into the credit. 
![image](https://github.com/PolicyEngine/policyengine-us/assets/113824411/94ae8014-6cb2-4442-8d7e-8b20d8659dfb)

Reply:
Summer credit hours do not count toward the semester requirement for credit hours. 
You may claim tuition paid for non-semester credit hours only if the student is seeking a degree from the school offering the course and if the student graduated from high school before the course began. A student seeking a degree at one college or university may transfer to another college or university in order to seek a degree and qualify at both schools.
The images were broken, here is the [link](https://dor.sc.gov/forms-site/Forms/I319_2021.pdf#page=3).